src/imxrt1180/lpuart.rs

```diff
@@ -0,0 +1,174 @@
+use super::pads::{
+    gpio_ad::*, gpio_aon::*, gpio_b1::*, gpio_b2::*, gpio_emc_b1::*, gpio_emc_b2::*, gpio_sd_b2::*,
+};
+use crate::{
+    consts::*,
+    lpuart::{Pin, Rx, Tx},
+    Daisy,
+};
+
+uart!(module: U1, alt: 0b0000, pad: GPIO_AON_08, direction: Tx, daisy: None);
+uart!(module: U1, alt: 0b0000, pad: GPIO_AON_09, direction: Rx, daisy: None);
+uart!(module: U2, alt: 0b0110, pad: GPIO_AON_01, direction: Rx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_RXD_SELECT_GPIO_AON_01));
+uart!(module: U2, alt: 0b0010, pad: GPIO_AON_27, direction: Rx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_RXD_SELECT_GPIO_AON_27));
+uart!(module: U2, alt: 0b0110, pad: GPIO_AON_00, direction: Tx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_TXD_SELECT_GPIO_AON_00));
+uart!(module: U2, alt: 0b0010, pad: GPIO_AON_26, direction: Tx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_TXD_SELECT_GPIO_AON_26));
+uart!(module: U3, alt: 0b0011, pad: GPIO_EMC_B1_02, direction: Rx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_RXD_SELECT_GPIO_EMC_B1_02));
+uart!(module: U3, alt: 0b0110, pad: GPIO_AD_14, direction: Rx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_RXD_SELECT_GPIO_AD_14));
+uart!(module: U3, alt: 0b0011, pad: GPIO_EMC_B1_03, direction: Tx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_TXD_SELECT_GPIO_EMC_B1_03));
+uart!(module: U3, alt: 0b0110, pad: GPIO_AD_13, direction: Tx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_TXD_SELECT_GPIO_AD_13));
+uart!(module: U5, alt: 0b0011, pad: GPIO_EMC_B1_15, direction: Rx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_RXD_SELECT_GPIO_EMC_B1_15));
```

Comment:
`GPIO_EMC_B1_15` is documented as `ALT2` in the reference manual. Since I can't always trust chapter 13, I cross checked against chapter 17. (`ALT3` would be ETH2_TX_CLK.)

```suggestion
uart!(module: U5, alt: 0b0010, pad: GPIO_EMC_B1_15, direction: Rx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_RXD_SELECT_GPIO_EMC_B1_15));
```

<img width="951" alt="image" src="https://github.com/user-attachments/assets/198ba8ff-4d63-4631-9864-6e9981fc90f9" />

Reply:
This is correctly an error I copied the wrong value from this part: 17.4.1.17 SW_MUX_CTL_PAD_GPIO_EMC_B1_15 SW MUX Control Register

![image](https://github.com/user-attachments/assets/46236f78-eeef-4740-bc42-a7207d2e5431)